### DISCUSSION TITLE
How using routes in vue components

### DISCUSSION THREAD

**Discussion by swarakaka** [adonisjs/core#4785] [Help] [answered]
Hello every one!

Does AdonisJS support using routes directly in Vue components with Inertiajs, similar to how [Ziggy](https://github.com/tighten/ziggy) works in Laravel?

**Comment by sinh117801** [comment, marked as answer]
I guess this one can help. https://packages.adonisjs.com/packages/izzyjs-route

**Reply by swarakaka** [reply]
Yes, that's it. thank you so much.
